infra/gcp/ensure-static-ips.sh

```diff
@@ -40,6 +40,8 @@ ADDRESSES=(
     "gcsweb-k8s-io"
     "k8s-io-ingress-canary"
     "k8s-io-ingress-prod"
+    "k8s-io-ingress-canary-v6"
```

Comment:
This calls ensure_global_address from infra/gcp/lib.sh, but that function doesn't understand v6.  Maybe we should just use the "-v6" suffix to tell that function to create IPv6 addresses?

Reply:
check if that is ok with you, my bash foo is not too great :)